app/experimenter/nimbus-ui/src/components/PageHome/DirectoryTable/index.tsx

```diff
@@ -288,10 +288,21 @@ export const DirectoryLiveTable: React.FC<DirectoryTableProps> = (props) => (
                 Looker
               </LinkExternal>
             )}
+            {experiment.monitoringDashboardUrl &&
+              experiment.resultsReady &&
+              experiment.rolloutMonitoringDashboardUrl && <br />}
```

Comment:
That's a bit specific, if we change those states around later this'll be awkward to keep in sync?  There's no simpler way to do that?

Reply:
This is just taken from below, line 302, with the addition of `rolloutMonitoringDashboardUrl`. I'm open to suggestions 😄

Reply:
Okay maybe let's just leave it and maybe @yashika-khurana can find a better way to do it when she tackles the sidebars.